doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -999,8 +999,9 @@ Documentation
 
 * Removed:
 
-  * Removed the Device configuration guides section and moved its contents to :ref:`ug_app_dev`.
+  * The Device configuration guides section and moved its contents to :ref:`ug_app_dev`.
   * The Advanced building procedures page and moved its contents to the :ref:`building` page.
+  * The Getting started with the nRF54L15 PDK page, and instead included the information about the `Quick Start`_ app support.
```

Comment:
We could have entry in the changelog for the structural changes of working with guide like we did for the other guides - https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/releases_and_maturity/releases/release-notes-2.7.0.html#working_with_nrf91_series. 

Also, information about the GSG could be moved to the Working with nRF54L Series section for more visibility?

Reply:
added